addons/medical/stringtable.xml

```diff
@@ -3460,7 +3471,7 @@
         </Key>
         <Key ID="STR_ACE_Medical_AdvancedMedicalSettings_painIsOnlySuppressed_Description">
             <English>Pain is only temporarily suppressed, not removed</English>
-            <German>Schmerzen werden nur vorübergehend unterdrückt, nicht entfernt</German>
+            <German>Schmerzen werden nur vorübergehend unterdrückt, nicht kuriert</German>
```

Comment:
`kuriert` klingt irgendwie nach 3 Monaten Urlaub in den Alpen. Könntest Du versuchen, dort ein ein besseres Wort zu finden?